pyiron_atomistics/atomistics/structure/neighbors.py

```diff
@@ -79,26 +107,63 @@ def __repr__(self):
             + "- distances : Distances to the neighbors of given positions\n"
             + "- indices : Indices of the neighbors of given positions\n"
         )
-        if self.vecs is not None:
+        if self._vecs is not None:
             to_return += "- vecs : Vectors to the neighbors of given positions\n"
         return to_return
 
     def copy(self):
         new_neigh = Tree(self._ref_structure)
-        new_neigh.distances = self.distances.copy()
-        new_neigh.vecs = self.vecs.copy()
-        new_neigh.indices = self.indices.copy()
+        new_neigh._distances = self._distances.copy()
+        new_neigh._indices = self._indices.copy()
         new_neigh._extended_positions = self._extended_positions
         new_neigh._wrapped_indices = self._wrapped_indices
-        new_neigh._allow_ragged = self._allow_ragged
         new_neigh._extended_indices = self._extended_indices
         new_neigh.wrap_positions = self.wrap_positions
         new_neigh._tree = self._tree
         new_neigh.num_neighbors = self.num_neighbors
         new_neigh.cutoff_radius = self.cutoff_radius
         new_neigh._norm_order = self._norm_order
+        new_neigh._positions = self._positions.copy()
         return new_neigh
 
+    def _reshape(self, value, key=None, ref_vector=None):
+        if key is None:
```

Comment:
This is absolutely a style thing, so ignore it if you don't like it! I've taken to one-lining this operation most of the time it pops up, i.e. `key = self.mode if key is None else key` (or the inverse `key = key if key is not None else self.mode`, personally I find them equally easy to read.) Ignore if you don't like.